rgdc/rgdc/types.py

```diff
@@ -0,0 +1,15 @@
+from datetime import datetime
+from typing import Literal, Tuple, Union
```

Comment:
`Literal` is introduced in Python 3.8: https://www.python.org/dev/peps/pep-0586/